components/authz-service/server/v2/policy_test.go

```diff
@@ -999,6 +999,14 @@ func TestUpdatePolicy(t *testing.T) {
 		{"fails with InvalidArgument when missing policy ID", func(t *testing.T) {
 			req := api_v2.UpdatePolicyReq{
 				Name: "testPolicy1",
+				Statements: []*api_v2.Statement{
```

Comment:
[nit] Guess we could have introduced a variable and re-used that instead of repeating the same block again and again

Reply:
That sounds worth it to me for the sake of test maintainability 🤷‍♂